website/static/js/paginator.js

```diff
@@ -4,8 +4,8 @@
 'use strict';
 var ko = require('knockout');
 var oop = require('js/oop');
-var MAX_PAGES_ON_PAGINATOR = 7;
-var MAX_PAGES_ON_PAGINATOR_SIDE = 5;
+var MAX_PAGES_ON_PAGINATOR = 16;
+var MAX_PAGES_ON_PAGINATOR_SIDE = 16;
```

Comment:
Why is `MAX_PAGES_ON_PAGINATOR` the same as `MAX_PAGES_ON_PAGINATOR_SIDE`? Shouldn't `MAX_PAGES_ON_PAGINATOR_SIDE` be less?